sympy/stats/joint_rv.py

```diff
@@ -289,7 +286,7 @@ class MarginalDistribution(Basic):
     distribution.
     """
 
-    def __new__(cls, dist, rvs):
+    def __new__(cls, dist, *rvs):
```

Comment:
Why this API change?

```python
def new(*a):
 return Tuple.fromiter(i for i in a)

def old(a):
 return Tuple.fromiter(i for i in a)

>>> new([1,2]))
([1, 2],)
>>> old([1,2]))
(1, 2)
```

Reply:
If this is the right API then the release notes should be updated. If an rv is never an iterable then this could be made compatible with the previous API by unpacking a singleton:

```python
if len(rvs) == 1 and iterable(rvs[0]):
    rvs = tuple(rvs[0])
```

Reply:
@czgdp1807 indeed, is it necessary to modify this line?

Reply:
Well, if we use the previous API, then we will pass parameters something like,

```
MGR = MarginalDistribution(B, (C[0],))
```

With the new API,
```
MGR = MarginalDistribution(B, C[0])
```
For me new API is more intuitive. The suggestion by @smichr can be applied to keep the previous API compatible.

Reply:
OK. We don't really need to keep the API compatible in this case, as the module was quite buggy and it's unlikely people were actually using it.